src/utils/github-utils.ts

```diff
@@ -239,14 +240,47 @@ const updateReadmeAndAutoMerge = async (repositoryOBJ: Repository, footer: strin
 export const updateReadmeAndAutoMergeRepositories = async (repositoriesOBJ: Repository[], footer: string) => {
   console.log("This script will add custom footers to ALL the specified repositories. It can also overwrite existing footers if desired.");
   console.log(chalk.red("If you do not want to automatically update ALL of the repositories selected, YOU MUST select 'n' in the following prompt."));
-  const processAllAnswer = promptSync(chalk.red("Do you want to add custom footer to ALL found repositories? (otherwise will be asked individually) (y/N): "));
+  
+  const processAllAnswer = promptSync(chalk.red("Do you want to add custom footer to ALL found repositories? (otherwise will be asked individually) (y/n/l for listing only repositories without a footer/q to quit): "));
```

Comment:
```suggestion
  const processAllAnswer = promptSync(chalk.red("Do you want to add custom footer to ALL found repositories? (otherwise will be asked individually) (y/n/l for listing only repositories without a footer, q to quit): "));
```

Reply:
Does this make more sense? Not sure why we would have a / there.